ajax/libs/cell/package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "cell",
+  "filename": "cell.min.js",
+  "description": "A self-constructing web app framework powered by a self-driving DOM",
+  "homepage": "https://www.celljs.org",
+  "license": "MIT",
+  "keywords": ["self-driving", "webapp", "framework"],
```

Comment:
Would you let the keyword format as same as others? This will helpful to us to maintain it.
Let's modify from A to B as the followings:

- A
```js
  "keywords": ["self-driving", "webapp", "framework"],
```

- B
```
  "keywords": [
    "self-driving",
    "webapp",
    "framework"
  ],
```